### DISCUSSION TITLE
Missing documentation

### DISCUSSION THREAD

**Discussion by redheadt** [timoschwarzer/gitlab-monitor#174] [Q&A]
Hello Timo, 
thank you for sharing this project!
I have tried to use your project but it was impossible for me. The reason for my failure might be, that I am a beginner in gitlab-ci. Nevertheless  in my opinion the documentation is not detailed enough. It would be very useful to me if the documentation was more detailed.

Thank you and best regards!

**Comment by timoschwarzer** [comment]
Hi!

I'm sure I can help if you tell me where you're stuck :)

**Comment by redheadt** [comment]
Thank you for your quick response!
I have downloaded your project and opend it in container through the implemented dockerfile. 
I tried yarn install ect. but the commands were not recognized. -> even if I installed yarn in my container. 

![screen2](https://user-images.githubusercontent.com/49395343/116078501-fc1d0e80-a696-11eb-845a-971f1ab645b1.png)

**Comment by timoschwarzer** [comment]
There is currently no Dockerfile for development. The Dockerfile you built is for production. It builds the application and bundles it into an nginx image (listens on port 80).
To start the application on your local machine for development purposes just run `yarn install` and `yarn dev` in your local terminal.

**Comment by redheadt** [comment]
Thank you!
It worked,  classical ISO/OSI layer 8 problem ;) 
While playing around I noticed that there is no general way which just shows the last pipeline of the last push no matter on which branch. To achieve this i could configure the working branch for each project manually but this is not maintainable. 

Have I overseen a setting or is this a "missing" or "unwanted" feature? 

Thanks again and all the best to you!

**Reply by timoschwarzer** [reply]
Do you mean showing the default branch only?

**Reply by redheadt** [reply]
No, I mean branch independent. I'd like to see the pipeline which got triggered at last but no matter which branch. 
Does this help you to understand my question?

**Reply by timoschwarzer** [reply]
Ah, gotcha! This is currently not possible afaik. If you want to, you can tinker around [here](https://github.com/timoschwarzer/gitlab-monitor/blob/main/src/components/project-card.vue#L210) to only get the branch with the latest commit. You can get the commit time with `branch.commit.committed_date` as it's done when sorting the branches a few lines below.
